pylint/checkers/base_checker.py

```diff
@@ -134,8 +134,8 @@ def get_full_documentation(
         if reports:
             result += get_rst_title(f"{checker_title} Reports", "^")
             for report in reports:
-                result += (
-                    ":%s: %s\n" % report[:2]  # pylint: disable=consider-using-f-string
+                result += ":{}: {}\n".format(  # pylint: disable=consider-using-f-string
```

Comment:
Why don't we use an `f-string` here?

Reply:
[6858e64](https://github.com/pylint-dev/pylint/pull/9086/commits/6858e6445a679d84dae26c4816b9d28e80e3dd4f)